PocketKit/Sources/Sync/CoreDataInitializers/Item+CoreDataProperties.swift

```diff
@@ -33,6 +33,8 @@ extension Item {
     @NSManaged public var recommendation: Recommendation?
     @NSManaged public var savedItem: SavedItem?
     @NSManaged public var syndicatedArticle: SyndicatedArticle?
+    @NSManaged public var collection: Collection?
+    @NSManaged public var collectionStory: CollectionStory?
```

Comment:
although `collectionStory` is not a property on `Item` in the graph, I followed what we are doing for slates and recommendations relationships